frontend/src/tests/components/eMIB/ResponseItem.test.js

```diff
@@ -0,0 +1,207 @@
+import React from "react";
+import { shallow } from "enzyme";
+import ResponseItem, { RESPONSE_TYPE } from "../../../components/eMIB/ResponseItem";
+import LOCALIZE from "../../../text_resources";
+
+it("renders the right response types (reply, reply all, forward)", () => {
+  //reply
+  const replyWrapper = shallow(
+    <ResponseItem
+      responseType={RESPONSE_TYPE.reply}
+      to={"to"}
+      cc={"cc"}
+      response={"response"}
+      reasonsForAction={"reasons"}
+    />
+  );
+
+  //reply all
+  const replyAllWrapper = shallow(
+    <ResponseItem
+      responseType={RESPONSE_TYPE.replyAll}
+      to={"to"}
+      cc={"cc"}
+      response={"response"}
+      reasonsForAction={"reasons"}
+    />
+  );
+
+  //forward
+  const forwardWrapper = shallow(
+    <ResponseItem
+      responseType={RESPONSE_TYPE.forward}
+      to={"to"}
+      cc={"cc"}
+      response={"response"}
+      reasonsForAction={"reasons"}
+    />
+  );
+
+  const reply = <span className="fas fa-reply" />;
+  const replyAll = <span className="fas fa-reply-all" />;
+  const forward = <span className="fas fa-share-square" />;
+
+  //reply
+  expect(replyWrapper.containsMatchingElement(reply)).toEqual(true);
+  expect(replyWrapper.containsMatchingElement(replyAll)).toEqual(false);
+  expect(replyWrapper.containsMatchingElement(forward)).toEqual(false);
+
+  //reply all
+  expect(replyAllWrapper.containsMatchingElement(reply)).toEqual(false);
+  expect(replyAllWrapper.containsMatchingElement(replyAll)).toEqual(true);
+  expect(replyAllWrapper.containsMatchingElement(forward)).toEqual(false);
+
+  //forward
+  expect(forwardWrapper.containsMatchingElement(reply)).toEqual(false);
+  expect(forwardWrapper.containsMatchingElement(replyAll)).toEqual(false);
+  expect(forwardWrapper.containsMatchingElement(forward)).toEqual(true);
+});
+
+it("renders email's header (with and without cc, since it is optional)", () => {
+  //with cc field
+  const withCcWrapper = shallow(
+    <ResponseItem
+      responseType={RESPONSE_TYPE.reply}
+      to={"to"}
+      cc={"cc"}
+      response={"response"}
+      reasonsForAction={"reasons"}
+    />
+  );
+
+  //without cc field
+  const withoutCcWrapper = shallow(
+    <ResponseItem
+      responseType={RESPONSE_TYPE.reply}
+      to={"to"}
+      response={"response"}
+      reasonsForAction={"reasons"}
+    />
+  );
+
+  const headerContentWithCc = (
```

Comment:
Can we simplify this to only check for `<span>cc</span>`?

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/172/commits/65fa676ee1553d366b684d85bae2a6d108d7851a